nengo/builder/connection.py

```diff
@@ -169,7 +171,7 @@ def get_prepost_signal(is_pre):
     # Sample transform if given a distribution
     transform = (conn.transform.sample(conn.size_out, conn.size_mid, rng=rng)
                  if isinstance(conn.transform, Distribution) else
-                 conn.transform)
+                 np.array(conn.transform))
```

Comment:
Ah yeah I see, you just moved the copy from line 204 to here.  So we'll end up with an extra copy of the transform in the `pre==Ensemble` case, but in that case the transform is probably pretty small so it doesn't really matter.